Documentation/custom_effects.md

```diff
@@ -48,6 +48,104 @@ These are some tips for writing or converting effects for use with MonoGame.
 * The effect compiler is aggressive about removing unused paramters, be sure the parameters you are setting are actually used.
 * If you think you've found a bug porting a shader [please let us know](https://github.com/mono/MonoGame/issues).
 
+### Summary of some work arounds in the process of transferring effects files across to MonoGame from XNA
+
+#### Aligning your version of the content processor and the runtime
+For shader .fx files, the version of the of the content processor you are using to compile your xnb files (ie. **MonoGameContentProcessors.dll**) **must be aligned** with the version of the runtime you are running your game against (ie. **MonoGame.Framework.dll**)
+
+Otherwise you may get **"Wrong MGFX file version!"** raised by ReadEffect() in Effect.cs when you do a **content.Load<Effect>**
+[because the version written in to the .xnb file is not a match with the value of MGFXVersion 
+that has been put in the run time].
+
+So if you have this error, the first thing to check is that your content processor is the same up-to-dateness as the version of the runtime you are running your game against.
+
+
+#### Shader model compilation:
+(a) For **OpenGL** versions of MonoGame (eg. Mac, Linus, WindowsGL) the shader model must be **SM 3.0 or lower**
+
+(b) For **DirectX** versions of MonoGame (eg. WinRT, Windows8) the shader model must be **SM 4.0 or higher**.
+
+
+
+In the DirectX case it is typical for people to use the:
+
+`vs_4_0_level_9_1`
+and
+`ps_4_0_level_9_1`
+
+compilation profiles, because these provide backwards compatibility for Shader Model 2 shaders.
+
+
+
+#### Windows8 configuration == DX11 (inside the content processor)
+(a) The "Windows8" build configuration is used internally in the MonoGame content processor code to indicate compilation to target DX11.
+
+Look at the code on line 29 of MGEffectProcessor.cs! 
+
+`options.DX11Profile = platform == MonoGamePlatform.Windows8 ? true : false; `
+
+
+
+The **ContentHelper.cs** file in the ContentProcessors project reads 
+the build configuration name you are using then does this:
+
+`switch (platform.ToUpper())`    
```

Comment:
When writing a block of code wrap it in the triple single quotes...  do not wrap each line individually.